### PR TITLE
Added select destination feature

### PR DESCRIPTION
**Pull request by syyang-hw**

### CODE REVIEW

File: Transfer/Sources/CreateTransferPresenter.swift [hunk 1]
```diff
@@ -63,6 +63,7 @@ final class CreateTransferPresenter {
     private var sourceToken: String?
 
     var selectedTransferMethod: HyperwalletTransferMethod!
+    var availableTransferMethods = [HyperwalletTransferMethod]()
```
Comment (jsaini-hw) [inline Transfer/Sources/CreateTransferPresenter.swift]: No need to add availableTransferMethods in here. Get it again from TransferMetthodRepository

File: Transfer/Sources/CreateTransferPresenter.swift [hunk 2]
```diff
@@ -218,11 +221,10 @@ final class CreateTransferPresenter {
 
     // MARK: - Destionation view
     func showSelectDestinationAccountView() {
-        // TODO Display all the destination accounts
-        //        view.showGenericTableView(items: getSelectDestinationCellConfiguration(),
-        //                                  title: "select_destination".localized(),
-        //                                  selectItemHandler: selectDestinationAccountHandler(),
-        //                                  markCellHandler: destinationAccountMarkCellHandler())
+        view.showGenericTableView(items: availableTransferMethods,
```
Comment (jsaini-hw) [inline Transfer/Sources/CreateTransferPresenter.swift]: get availableTransferMethods from TransferMethodRepository

**Comment by jsaini-hw** [comment]
Add unit test for showSelectDestinationAccountView

File: Transfer/Sources/CreateTransferPresenter.swift (hunk 2 above)
Comment (syyang-hw) [reply Transfer/Sources/CreateTransferPresenter.swift]: Sure

**Comment by syyang-hw** [comment]
> Add unit test for showSelectDestinationAccountView

I would merge this first and then add ui tests later because CreateTransferPresenterTests does not exist yet, and Yurii might be working on the tests as well.

File: Transfer/Sources/CreateTransferPresenter.swift [hunk 3]
```diff
@@ -219,11 +218,25 @@ final class CreateTransferPresenter {
...
+                    strongSelf.view.showGenericTableView(items: transferMethods,
+                                                         title: "select_destination".localized(),
+                                                         selectItemHandler: strongSelf
+                                                            .selectDestinationAccountHandler(),
+                                                         markCellHandler: strongSelf
+                                                            .destinationAccountMarkCellHandler())
+                }
+
+            case .failure:
+                break
```
Comment (jsaini-hw) [inline Transfer/Sources/CreateTransferPresenter.swift]: Add error handling in here

**Pull request merged**